src/server_manager/electron_app/build_action.sh

```diff
@@ -40,13 +40,7 @@ cp -r $BUILD_DIR/server_manager/web_app/static $STATIC_DIR/server_manager/web_ap
 cp $MODULE_DIR/loading.html $STATIC_DIR/server_manager/web_app/
 
 # Electron requires a package.json file for the app's name, etc.
-# We also need to install NPMs at this location for require()
```

Comment:
I don't understand why we needed this install before, so I can't tell the change is safe.

What require did we have before?
Didn't browserify used to remove them already?

Reply:
We used require in HTML/js before. I think an earlier change of yours removed that. I wasn't sure how safe the change is, but nothing broke when testing the app.